reference/orders.v2.oas2.yml

```diff
@@ -3929,9 +3931,21 @@ components:
           example: '0.0000'
           type: string
         ip_address:
-          description: 'IP Address of the customer, if known.'
+          type: string
+          description: |-
+            IPv4 Address of the customer, if known.
+
+            Note: either IPv4 or IPv6 can be set. Setting IPv4 value will reset the IPv6 value and vice versa
```

Comment:
I think it would be more clear to say:
either **ip_address** or **ip_address_v6** can be set. Setting **ip_address** value will reset the **ip_address_v6** value and vice versa.

Reply:
good point, doc updated 🙇

Reply:
Thanks 🙇‍♀️